deploy/helm_charts/dc_website/templates/ingress.yaml

```diff
@@ -13,17 +13,6 @@
 # limitations under the License.
 
 {{- if .Values.ingress.enabled }}
-apiVersion: networking.gke.io/v1
```

Comment:
why is this removed?

Reply:
Currently certificate is created in Terraform, this one was never used (nowherre in our code references the name).